VirtoCommerce.Storefront.Model/Catalog/Product.cs

```diff
@@ -389,7 +389,9 @@ public void ApplyTaxRates(IEnumerable<TaxRate> taxRates)
 
         public void ApplyRewards(IEnumerable<PromotionReward> rewards)
         {
-            var productRewards = rewards.Where(r => r.RewardType == PromotionRewardType.CatalogItemAmountReward && (r.ProductId.IsNullOrEmpty() || r.ProductId.EqualsInvariant(Id)));
+            var productRewards = rewards.Where(r => r.RewardType == PromotionRewardType.CatalogItemAmountReward && (r.ProductId.IsNullOrEmpty() || r.ProductId.EqualsInvariant(Id))
```

Comment:
We do not need these rules for the product, because there is no quantity

Reply:
If we have condition "Get 50% off for 1 product1 for every 1 of product1" or "Get 50% off for 1 product1 for every 1 of product2" - don't we need to show discounted product price (for 1 entry, on page on screenshot)?
![image](https://user-images.githubusercontent.com/15610352/50469131-f59a0180-09cc-11e9-984b-e00115ce4cf7.png)

Reply:
Fixed.